src/module-elasticsuite-catalog-graph-ql/DataProvider/Product/LayeredNavigation/Builder/Attribute.php

```diff
@@ -113,7 +113,7 @@ function ($frontendLabel) use ($storeId) {
             }
 
             $isManualOrder = $attribute->getFacetSortOrder() == ElasticBucketInterface::SORT_ORDER_MANUAL;
```

Comment:
`$attribute` will be null if the `$attributeCode` is `attribute_set_id` : the load in line 86 will not work and we'll end up in the catch line 96.

In this case, we cannot call `$attribute->getFacetSortOrder()`. We must check the attribute code before.

As suggested by the users, the lines should become this : 

```
if ($attributeCode !== 'attribute_set_id' &&
    $attribute->getFacetSortOrder() == ElasticBucketInterface::SORT_ORDER_MANUAL) {
```